CHANGELOG.md

```diff
@@ -3,15 +3,23 @@
 ## 3.0
 
 ### NEXT
-* performance optimization: Instantiate fewer KmmResults
 
+* Fix ASN.1 decoding flaw for a very specific length encoding
+* Performance optimization: Instantiate fewer KmmResults
 * Move `PemEncodable`/`PemDecodable` from _indispensable_ to _indispensable-asn1_ module.
+* More comprehensive PEM encoding/decoding support:
+    * `CryptoPublicKey`
+      * Note that PKCS1 encoding of RSA keys is not supported as it is discouraged (decoding is supported)
```

Comment:
i still think we should implement this if it's not too much effort (same way we do it for private keys w/ named member objects)

Reply:
then file an enhancement issue

Reply:
is it that complex that we couldn't do it here?

Reply:
if so, fine, please open the tracker issue and reference it here

Reply:
see #237